heisskleber/mqtt/publisher_async.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+import asyncio
+from typing import Any
+
+import aiomqtt
+
+from heisskleber.core.packer import get_packer
+from heisskleber.core.types import AsyncSink
+
+from .config import MqttConf
+
+
+class AsyncMqttPublisher(AsyncSink):
```

Comment:
naming collision: pubplisher vs sink

Reply:
A publisher is a type of sink (A sender would also be a sink). Since mqtt is using a pub / sub model, it'd still refer to the class as a publisher. At least that's my line of reasoning.

Reply:
Alright with me! :)